packages/toml/pkg/covector_toml_bg.wasm.d.ts

```diff
@@ -0,0 +1,16 @@
+/* tslint:disable */
+/* eslint-disable */
+export const memory: WebAssembly.Memory;
```

Comment:
I think we want to `.gitignore` this `pkg` directory, and allow it to be committed in the `action`. We don't _need_ it here, ultimately only in the `action` directory.